checks/rabbitmq/check.go

```diff
@@ -78,7 +78,7 @@ func New(config Config) func() error {
 			return err
 		}
 
-		if _, err := ch.QueueDeclare(config.Queue, false, false, false, false, nil); err != nil {
+		if _, err := ch.QueueDeclare(config.Queue, false, true, false, false, nil); err != nil {
```

Comment:
We already had setup like this - it does not work causing an application to crash because of the race condition. If you have two instances the first one consumes a message and the queue gets deleted while the second instance tries to bind the queue that is already deleted.

With persistent queue you always has one queue available. If you need to remove queues you should do it explicitly and either manually or semi-manually.